spec/contracts.dd

```diff
@@ -99,18 +97,22 @@ body
     return cast(long)std.math.sqrt(cast(real)x);
```

Comment:
And here they are:
https://github.com/dlang/dlang.org/pull/1828
https://github.com/dlang/dlang.org/pull/1829